### ISSUE TITLE
OneGet Integration

### ISSUE THREAD

**Issue by flofreud** [psget/psget#138]
With the release of powershell 5.0 sometimes in the future, there is a new toolset called OneGet. Probably a PsGet provider could be written for OneGet: http://learn-powershell.net/2014/04/03/checking-out-oneget-in-powershell-v5/

**Comment by angellaa** [comment]
I was just in the process of calling "Install-Module posh-git" on Windows 10 (as it has OneGet by default).... yes it would be useful :+1:

**Comment by flofreud** [comment]
It could be that PsGet will be obsolete with Windows 10. OneGet seems to contain a component for installation of PowerShell modules: https://www.powershellgallery.com/

**Comment by chaliy** [comment]
Yes, it is obsolete now, I am going to update pages and tool itself with notices.

**Comment by lboening** [comment]
I would like to see PSGet continue. I cannot easily add private repositories. Am I missing something with OneGet? It is Nuget-oriented.

**Comment by lzybkr** [comment]
@lboening - Private repositories are a supported scenario for OneGet, see - http://blogs.msdn.com/b/powershell/archive/2014/05/20/setting-up-an-internal-powershellget-repository.aspx
